### PR TITLE
Task/create user form

### PR DESCRIPTION
**Pull request by csssaz**
#### What's this PR do?
It implements the (Django)Form to create users inside the administrador app.

#### Where should the reviewer start?
administracion/forms.py
administracion/tests.py
jp2_online/urls.py

#### How should this be manually tested?
python manage.py test administracion

#### Any background context you want to provide?
Please ignore the changes shown in perfiles_usuario, since they are in other PR. (They will be merged with this one once both are accepted)

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: administracion/templates/crear_usuario.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (erikiado) [inline administracion/templates/crear_usuario.html]: If this file is used, it should be on: jp2_online/templates/administracion/

File: administracion/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from .views import crear_usuario
+
+app_name = 'administracion'
```
Comment (erikiado) [inline administracion/urls.py]: This variable is not used, it should be removed

File: perfiles_usuario/tests.py [hunk 3]
```diff
@@ -1,3 +1,145 @@
...
+from django.contrib.auth.models import Group
 
-# Create your tests here.
+from .utils import is_member, is_administrador, is_capturista, is_directivo, is_servicios_escolares
+from .utils import ADMINISTRADOR_GROUP, CAPTURISTA_GROUP, \
+                DIRECTIVO_GROUP, SERVICIOS_ESCOLARES_GROUP
+from .models import Capturista
+
+
+class PermisosTestCase(TestCase):
```
Comment (erikiado) [inline perfiles_usuario/tests.py]: Documentation for the class is needed. The method setUp also needs documentation.

Comment (csssaz) [inline perfiles_usuario/tests.py]: It's on PR #10

**Pull request merged**